README.md

```diff
@@ -51,29 +51,25 @@ yarn add markdown-to-jsx
 
 `markdown-to-jsx` exports a React component by default for easy JSX composition:
 
-ES6-style usage\*:
-
 ```jsx
 import Markdown from 'markdown-to-jsx';
 import React from 'react';
 import {render} from 'react-dom';
 
-const markdown = `# Hello world!`.trim();
-
 render((
-    <Markdown>
-        # Hello world!
-    </Markdown>
-), document.body);
+    <Markdown>{`
 
-/*
-    renders:
+# Hello world!
 
-    <h1>Hello world!</h1>
- */
+    `}</Markdown>
+), document.body);
 ```
 
-\* __NOTE: JSX does not natively preserve newlines in multiline text. In general, writing markdown directly in JSX is discouraged and it's a better idea to keep your content in separate .md files and require them, perhaps using webpack's [raw-loader](https://github.com/webpack-contrib/raw-loader).__
```

Comment:
That example does work, but I want to leave in this note because it will save people a lot of time if they don't use the exact syntax.

Reply:
https://github.com/probablyup/markdown-to-jsx/pull/155/commits/65aa3ee8b5d93f1630ba0d15fbbfa661d0dc0742